_posts/2015-01-01-app-review-process.md

```diff
@@ -55,19 +56,18 @@ Developer returns completed and signed forms to the Poynt app team by responding
 
 
 ### Step 8:
-At this point, the **Poynt App Review Team** is notified and an app review analyst reviews the submitted app.
-During the review if any issues are found the app is rejected with the recommendation to update the app for further consideration. The app is placed back in the queue(Step 2) if rejected.
-If the app passes all checks, the app moves to the next step in the process.
+When an app is submitted, Poynt app analyzer reviews the app within 24 hours. If any issues are found the app is rejected with the recommendation to update the app for further consideration. The results are made available under the developer portal for review. The app is placed back in the queue(Step 2) if rejected.
```

Comment:
Don't use 'poynt app analyzer' since it's an internal tool people don't need to know about. Just say the app will be reviewed within 24 hours. Should add a screenshot of a rejected app with reasons.